### DISCUSSION TITLE
Geomap Multiple Marker Layers

### DISCUSSION THREAD

**Discussion by mertbakir** [grafana/grafana#45756] [Dashboards, Panels & Visualizations]
First of all, I really liked the developments on the Geomap panel. Big thanks to the devs.

My question is, I couldn't find a way to separate or filter data for multiple marker layers.
Example:
- I have 50 rows with latitude and longitude and a category field. 
- Let's say 20 of 50 with Category = A, 30 is Category = B.
- I want to use triangle.svg symbol for the first category and circle for the other category.

Is this not currently possible, or am I missing something?

**Comment by nmarrs** [comment]
Hi @mertbakir, thanks for raising this up. I looked into this and it appears that we currently do not support the ability to filter markers from the same query, though it does seem reasonable for us to put on our [Geomap backlog](https://github.com/grafana/grafana/issues/44240) to consider as an ease of use addition.

You should always be able to split data into two separate queries (by category in this case) and effectively filter by category that way
<img width="1510" alt="filter via 2 query example" src="https://user-images.githubusercontent.com/22381771/177429617-68373431-aa0c-4dd8-b12c-90d9040b0678.png">

Another potential alternative might be labeling your marker via your desired field

https://user-images.githubusercontent.com/22381771/177428629-d0177468-8aa1-45ba-b9c6-17e0ce9ac7a8.mov

**Reply by mindraisin** [reply]
why does the 'Data' dropdown under 'Layer type' is not there in my edit panel?

**Reply by nmarrs** [reply]
Hi @mindraisin, this could be due to not being on the latest version of Grafana as the Data dropdown was added in a more recent version

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
